python/pyarrow/table.pxi

```diff
@@ -5364,30 +5364,31 @@ list[tuple(str, str, FunctionOptions)]
         ----
         keys: [["a","b","c"]]
         """
-        columns = [a[0] for a in aggregations]
+        target_cols = [a[0] if isinstance(a[0], (list, tuple)) else [a[0]] for a in aggregations]
```

Comment:
I don't know if we want to be pythonic and accept any iterable here or not.  However, I suppose we're pretty strict elsewhere.  Can you update the example in the docstring with a call to `count(*)`?

CC @amol- or @jorisvandenbossche for canonical advice

Reply:
I will update the example and Python help is very welcome! I tried to copy patterns I saw elsewhere in this codebase because I can't claim I'm able to write pythonic Python :)